data/human/hails.txt

```diff
@@ -3694,38 +4178,401 @@ phrase "hostile pirate"
 		" is the best way to live!"
 		" is how real anarchists live!"
+		" is how all men should live!"
 
 phrase "hostile pirate"
 	word
-		"You're about to "
-		"Get ready to "
-		"I'm going to make you "
-		"I hope you're ready to "
-		"Unless you pay me a hefty bribe, you're going to "
-		"I'm here to make you "
-		"I'm here to see to it that you "
-		"Pay up or "
-		"Hand over your credits or get ready to "
-		"Gimmie yer money or "
-	word
-		"die!"
-		"eat vacuum!"
-		"eat flaming death!"
-		"get blown up!"
-		"turn into a lovely cloud of rapidly expanding debris!"
-		"die an embarrassing death!"
-		"get turned into space debris!"
-		"eat cold hard vacuum!"
-		"die a horrific death!"
-		"get boarded and plundered!"
-		"contribute your wealth to furthering the cause of anarchy!"
-		"wind up a lot poorer!"
-		"get sent to Davy Jones' locker!"
+		"I'm continuing"
+		"I'm proudly continuing"
+		"I'm a contributor towards"
+		"I'm contributing to"
+		"I'm preserving"
+		"I'm maintaining"
+		"I'm proudly maintaining"
+		"I'm promoting"
+		"I'm espousing"
+		"I'm supporting"
+		"I'm assisting"
+		"I'm fighting for"
+		"I'm furthering"
+		"I'm here to further"
+		"I'm seizing your cargo to further"
+		"I'm here to make you a part of"
+		"I'm committing myself to"
+		"I'll gladly sacrifice you for"
+		"I'll make sure you get destroyed for"
+		"I'll make your death a contribution towards"
+		"I'll make this another victory for"
+		"You're the next victim of"
+		"You'll be the next to die in the name of"
+	word
+		" the "
+		" our "
+	word
+		"great"
+		"grand"
+		"noble"
+		"dignified"
+		"rebellious"
+		"defiant"
+		"obstinate"
+		"unyielding"
+		"lovely"
+		"bloody"
+		"underappreciated"
+		"necessary"
+		"inevitable"
+		"five-hundred year old"
+		"thousand year old"
+		"millennia-long"
+		"eons-long"
+		"long living"
+		"inextinguishable"
+		"eternal"
+		"undying"
+		"unceasing"
+		"indestructible"
+		"endless"
+	word
+		" "
+	word
+		"tradition"
+		"lifestyle"
+		"convention"
+		"custom"
+		"practice"
+		"heritage"
+		"movement"
+		"code"
+		"cause"
+		"ideals"
+		"ideology"
+		"struggle"
+	word
+		" of "
+	word
+		"blowing spaceships up for a living"
+		"blowing merchants up for a living"
+		"destroying oppressors for a living"
+		"taking from the rich and giving to the poor"
+		"using force to make peace"
+		"abstinence from taxes"
+		"anarchism"
+		"piracy"
+		"being the equaliser between men"
+		"equality without government"
+		"life without taxes"
+		"making death the only constant of life"
+		"life without oppressors"
+		"freedom"
+		"liberating those who don't know they need to be liberated"
+		"saving the population from themselves"
+		"painting my ship red instead of blue"
+		"shooting first and asking questions never"
+		"raiding merchant convoys"
+		"defeating the Law"
+		"defeating the Man"
+		"making ${drugs} legal"
+		"collectivism"
+		"freedom from oppressors"
+		"living without authority"
+	word
+		"!"
```

Comment:
The drugs phrase contains a few words with `%` at the end, expecting it to be replaced depending on if you're looking for plural or singular names for the drugs. In this case it's singular, so we only need to replace any `%` with nothing.
```suggestion
		"!"
	replace
		"%" ""
```